liferay-workspace/apps/service-builder/basic/basic-service/service.xml

```diff
@@ -1,7 +1,7 @@
 <?xml version="1.0"?>
-<!DOCTYPE service-builder PUBLIC "-//Liferay//DTD Service Builder 7.1.0//EN" "http://www.liferay.com/dtd/liferay-service-builder_7_1_0.dtd">
+<!DOCTYPE service-builder PUBLIC "-//Liferay//DTD Service Builder 7.2.0//EN" "http://www.liferay.com/dtd/liferay-service-builder_7_2_0.dtd">
 
-<service-builder package-path="com.liferay.blade.samples.servicebuilder">
+<service-builder dependency-injector="ds" package-path="com.liferay.blade.basic">
```

Comment:
Is this the default or is spring the default?

Reply:
So, we dont actually need `dependency-injectory="ds"` since ds is default, but i just wanted to somehow show that it's configurable.